x-pack/plugins/observability_solution/infra/public/components/asset_details/tabs/logs/logs.tsx

```diff
@@ -106,18 +107,12 @@ export const Logs = () => {
             />
           </EuiFlexItem>
           <EuiFlexItem grow={false}>
-            <EuiButtonEmpty
-              data-test-subj="infraAssetDetailsLogsTabOpenInLogsButton"
+            <OpenInLogsExplorerButton
+              href={logsUrl}
```

Comment:
> [!WARNING]
> This `logsUrl` is created using the `nodeLogsLocator` locator, but I see you removed this locator. Is this link working as expected?

Reply:
Yep 👍 The `NODE_LOGS_LOCATOR_ID` is registered in logs shared, and is ultimately translated to the `ALL_DATASETS_LOCATOR` (from logs explorer).